packages/firestore/test/lite/integration.test.ts

```diff
@@ -106,14 +106,56 @@ describe('Firestore', () => {
     expect(fs1).to.be.an.instanceOf(Firestore);
   });
 
-  it('returns same instance', () => {
+  it('returns same default instance from named app', () => {
```

Comment:
Please also add some tests to `test/integration/api/database.test.ts`. No need to be this thorough, just sanity checks.

Reply:
Add tests. IN doing so, some problem were revealed. Fixed.